src/main/kotlin/io/github/mojira/arisa/registry/ModuleRegistry.kt

```diff
@@ -63,19 +92,20 @@ abstract class ModuleRegistry(private val config: Config) {
         moduleName to tryExecuteModule { module(issue, lastRun) }
     }
 
-    private fun getJqlWithDebug(lastRun: Instant): String {
+    private fun getJqlWithDebug(timeframe: TicketQueryTimeframe): String {
+        val registryJql = getJql(timeframe)
+
         val debugWhitelist = config[Arisa.Debug.ticketWhitelist]
 
-        return if (debugWhitelist == null) getJql(lastRun)
-        else "key IN (${debugWhitelist.joinToString()}) AND (${getJql(lastRun)})"
+        return if (debugWhitelist == null) registryJql
+        else "key IN (${debugWhitelist.joinToString()}) AND ($registryJql)"
     }
 
-    fun getFullJql(lastRun: Instant, failedTickets: Collection<String>): String {
-        val projects = config[Arisa.Issues.projects]
-        val failedTicketsJql = if (failedTickets.isEmpty()) "key in (${ failedTickets.joinToString() })" else ""
-        val registryJql = "(${ getJqlWithDebug(lastRun) })"
+    fun getFullJql(timeframe: TicketQueryTimeframe, failedTickets: Collection<String>): String {
+        val failedTicketsJql = if (failedTickets.isEmpty()) "" else "key in (${ failedTickets.joinToString() }) OR "
+        val registryJql = "(${ getJqlWithDebug(timeframe) })"
 
-        return "project IN (${ projects.joinToString() } AND ($failedTicketsJql OR $registryJql) ORDER BY updated ASC"
```

Comment:
Did you remove `project IN ...` on purpose?

Reply:
Ah, yes, I did. Reason for this is that this caused incorrect behaviour; Arisa also checks tickets that are not in the global config, weirdly enough. Most noticeably, this would have completely disabled the ResolveTrashModule.

Reply:
Note that I added it in the previous commit(s), it was not originally there.

Reply:
Ah thanks for the clarification. It looks like module specific project lists take precedence:
https://github.com/mojira/arisa-kt/blob/253faaa191b0c3b34a23493e28a09661a9baf54c/src/main/kotlin/io/github/mojira/arisa/ModuleExecutor.kt#L114